src/parser/test/ParserTest.cpp

```diff
@@ -227,6 +227,20 @@ TEST(Parser, SpaceOperation) {
         auto result = parser.parse(query);
         ASSERT_TRUE(result.ok()) << result.status();
     }
+    {
+        GQLParser parser;
+        std::string query = "CREATE SPACE default_space(partition_num=9, replica_factor=3,"
+                            "atomic_edge=true)";
```

Comment:
Isn't the transaction on edge operation supported by default? Why need user to specify it when creating space?

Reply:
txn will disable by default (because of performance?)

Reply:
I have the same doubts. I think the consistency guarantee is a MUST we have to provide. Otherwise, it will result in dirty data.

Reply:
seems we should ensure the transaction when writing in and out edges ?

Reply:
Previously data is not guaranteed to be consistent, so as for use update from 1.0, enable toss by default can't fix previous inconsistency. If a new space, enable toss sounds reasonable.

On the other hand, we'd better test the whole 2.0 system when toss enabled, both for stability and performance before it is released officially. That will be a good time to enable it by default.

Reply:
> enable toss by default can't fix previous inconsistency

We can do nothing about the old dirty data anyway while it does not affect the fact that we _enable_ the so called toss by default. Thus I think it's even not necessary to introduce such an option.

Reply:
> Thus I think it's even not necessary to introduce such an option.

Agree. After it is stable, we could remove it.

Reply:
Fair enough